conf/mig-agent.cfg.inc

```diff
@@ -54,15 +54,20 @@
     ; if true, only the investigator's public key is verified on actions and not ACLs.
     onlyVerifyPubKey = false
 
+    ; Tag the instance with attributes to filter investigation scope
+    tags = "operator:Ops_Emea"
+    tags = "role:web"
+
+
 [stats]
     ; number of previous actions the agent should keep a summary for. the summary can
     ; be viewed over the agent stat socket. 0 to disable.
     maxactions = 15
 
-[certs]
-    ca  = "/path/to/ca/cert"
-    cert= "/path/to/client/cert"
-    key = "/path/to/private/key"
+; [certs]
+;    ca  = "/path/to/ca/cert"
+;    cert= "/path/to/client/cert"
+;    key = "/path/to/private/key"
```

Comment:
That was intentional, I copy over the config in `tools/standalone_install.sh`. I could make the standalone_install comment out those lines instead of editing the template.